sshbuf-getput-crypto.c

```diff
@@ -143,7 +145,7 @@ sshbuf_put_bignum2(struct sshbuf *buf, const BIGNUM *v)
 }
 
 int
-sshbuf_put_ec(struct sshbuf *buf, const EC_POINT *v, const EC_GROUP *g)
+sshbuf_put_ecbuf(struct sshbuf *buf, const EC_POINT *v, const EC_GROUP *g)
```

Comment:
I don't understand the new names for the functions and would have kept the old ones